cmd/deploy/deploy.go

```diff
@@ -656,6 +677,37 @@ func (dc *DeployCommand) cleanUp(ctx context.Context) {
 	}
 }
 
+func (dc *DeployCommand) createTempOktetoEnvFile() (afero.File, error) {
+	oktetoEnvFileDir, err := afero.TempDir(dc.Fs, "", "")
+	if err != nil {
+		return nil, err
+	}
+
+	oktetoEnvFile, err := dc.Fs.Create(fmt.Sprintf("%s/.env", oktetoEnvFileDir))
```

Comment:
Should we convert `.env` to a constant?

Reply:
It would only be used here by aasique... I would say no